operations/network/management_network/bican_technical_details.md

```diff
@@ -0,0 +1,312 @@
+- **Add** migrate administrative traffic from CAN to the new Management CAN (CMN).
+
+**Reverting or changing any decisions will be manual**
+
+The feature matrix of the CAN is described [here](bican_support_matrix.md).
+Details of the High Speed CAN (CHN) and the Management CAN (CMN) are described below.
+
+![Customer Access Overview](img/customer_access_overview.png)
+
+<a name="2-high-speed-can-(chn)"></a>
+
+## 2 High Speed CAN (CHN)
+
+Access to system resources from the customer site over the High Speed Network is provided by the High Speed CAN (CHN).
+As can be seen in the diagram above, traffic ingress from the site for the CHN is over the Edge Routers
+(typically a pair of Arista Switches which provide other HSN access - for ClusterStor for instance).
+
+NOTE:  Arista routing configurations as a virtual routing instance are in-scope for CSM 1.2 CHN work.
+Other Edge Routers, including Juniper are out of scope.
+
+<a name="21-chn-system-ingress-endpoints-accessible-in-CSM 1.2"></a>
+
+### 2.1 CHN system ingress endpoints accessible in CSM 1.2
+
+- Designated Application Nodes, particularly **UAN, over SSH**.
+- Designated **Compute Nodes (CN)**, including those used for Compute as UAN, **over SSH**.
+- Kubernetes **API endpoints over https**.
+
+<a name="22-chn-system-egress-endpoints-accessible-in-CSM 1.2"></a>
+
+### 2.2 CHN system egress endpoints accessible in CSM 1.2
+
+- System access to **site external resources** , including LDAP(s) and DNS.
+
+<a name="23-endpoint-naming"></a>
+
+### 2.3 Endpoint Naming
+
+A ".CHN" DNS suffix will be used for all endpoints accessed over the High Speed CAN.
+Endpoints naming will be resolved and maintained in the system Authoritative DNS (another CSM 1.2 feature).
+As part of the introduction of authoritative DNS endpoints will also have a top-level-domain appended, creating a fully qualified domain system.
+
+Examples:
+
+- `uan01.chn.tld` as resolved externally.
+- `uan01.chn` can be resolved internal to the system (maintained via local `resolv.conf`).
+- `nid000001.chn.tld`
+- `api-gateway-service.chn.tld`
+
+Where `tld` is configurable at installation and can be a subdomain of the site domain system.
+Exchange of system DNS with the site may be via delegation (preferred) or zone transfer (AXFR).
+
+Once added to CSI, names and IP's will use the standard CSM data flow and end up in SLS and be available for use via both DNS and DHCP services.
+
+<a name="231-touchpoints:-effects-and-changes"></a>
+
+#### 2.3.1 Touchpoints: effects and changes
+
+- Will require installation and administration document changes.
+- The CHN requires a small change in CSI to add this network.
+- Will (automatically) use the DNS infrastructure from previous CSM install.
+- Name aliases can be added/changed/removed via the API to SLS and become available in DNS automatically.
+DNS tooling for this was released in Shasta v1.4 with [SLS](../../index.md#system-layout-service-sls)
+
+<a name="232-when-naming-occurs"></a>
```

Comment:
Note that for main (since I assume this is getting a main backport) that all of these anchors refs are no longer.

I'd suggest using the actual headers in this PR to make the backport simpler.

Reply:
removed <a> anchor links and updated page links